be/src/olap/rowset/beta_rowset_writer.cpp

```diff
@@ -98,29 +98,34 @@ Status BetaRowsetWriter::init(const RowsetWriterContext& rowset_writer_context)
 }
 
 Status BetaRowsetWriter::add_block(const vectorized::Block* block) {
```

Comment:
Is there any function call add_block? If not, just remove this method.

Reply:
checkpoint should call the function.